storage/aptosdb/src/state_store/state_store_test.rs

```diff
@@ -287,7 +292,43 @@ pub fn test_get_state_snapshot_before() {
     assert_eq!(store.get_state_snapshot_before(3).unwrap(), Some((2, hash)));
     assert_eq!(store.get_state_snapshot_before(2).unwrap(), Some((0, hash)));
     assert_eq!(store.get_state_snapshot_before(1).unwrap(), Some((0, hash)));
-    assert_eq!(store.get_state_snapshot_before(0).unwrap(), None,);
+    assert_eq!(store.get_state_snapshot_before(0).unwrap(), None);
+
+    // Only consider a version as available when the root node is there.
```

Comment:
Comment a bit more on why you put in another node.

Reply:
Done